Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM openjdk:19-jdk-alpine
+FROM alpine
 COPY target/scala-3.3.3/dr2-disaster-recovery.jar /dr2-disaster-recovery.jar
-RUN mkdir -p /poduser/work /poduser/repo /poduser/version && \
+RUN apk update && apk upgrade && apk add openjdk19-jre && \
```

Comment:
Could we not use a newer version of of openjdk?

Reply:
I was using the latest version of openjdk that would work nicely with Scala. The base alpine image is always going to get the fixes first anyway and there's no downside to installing the JRE ourselves.